Functions.Templates/Templates/SqlOutputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async function (context, req) {
```

Comment:
`// Visit https://aka.ms/sqlbindingsoutput to learn how to use this output binding`